package.json

```diff
@@ -45,7 +45,7 @@
     "eslint-plugin-prettier": "^3.3.1",
     "eslint-plugin-react": "^7.22.0",
     "eslint-plugin-react-hooks": "^4.2.0",
-    "husky": "^4.3.8",
+    "husky": "^5.0.9",
```

Comment:
Let's not push the code which is not required for the current issue we are solving.
But as this is a small change and not affecting much, we can keep this change for now.

Let's keep this in mind from future 😃

Reply:
Actually, I was facing an issue while committing it.
So in order to fix the commit issue, I had to update it

Reply:
What issue, can you please explain here?

Reply:
it was related to the pre-commit hook

**pre-commit hook failed (add --no-verify to bypass)**

[https://github.com/typicode/husky/issues/716](https://github.com/typicode/husky/issues/716)